App/Observables/ViewModels/AccountViewModel.swift

```diff
@@ -13,7 +13,7 @@ import Foundation
 /// for KronoX events, and booking and unbooking of resources.
 final class AccountViewModel: ObservableObject {
     let viewModelFactory: ViewModelFactory = .shared
-    let userController: UserController = .shared
+    @Published var userController: UserController = .shared
```

Comment:
This should not be a published value. Either shared singleton or make it a dependency we load on boot